client/blocks/author-selector/switcher-shell.jsx

```diff
@@ -132,24 +127,24 @@ const SwitcherShell = createReactClass( {
 				</Popover>
 			</span>
 		);
-	},
+	}
 
-	_isLastPage: function() {
+	_isLastPage() {
```

Comment:
What do you think to taking this opportunity to rename methods like this that are prefixed with `_`?
I'm totally fine with skipping them for now, but our guides advise against such naming. ([ref](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#react-components) )

Reply:
Thanks for a good idea @spen, I removed all the underscores from method names, too.